<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bookcar</class>
 <widget class="QDialog" name="bookcar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1315</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="bookCarPicLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1321</width>
     <height>711</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../pexels-pixabay-33488.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>671</width>
     <height>661</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:grey;
border:5px solid black;
border-radius:8px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitButton">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>630</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:red;
border-radius:10px;
border:3px solid black;</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BookCarButton">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>630</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:green;
border-radius:12px;
border:3px solid black;</string>
   </property>
   <property name="text">
    <string>Book Car</string>
   </property>
  </widget>
  <widget class="QLabel" name="addCarLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>22</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:yellow;</string>
   </property>
   <property name="text">
    <string>Book Car</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>161</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;</string>
   </property>
   <property name="text">
    <string>Name : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NameLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="AgeLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="ageLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>161</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;</string>
   </property>
   <property name="text">
    <string>Age :</string>
   </property>
  </widget>
  <widget class="QLabel" name="GenderLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>161</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;</string>
   </property>
   <property name="text">
    <string>Gender :</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="MaleRadioButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>260</y>
     <width>110</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Male</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="FemaleRadioButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>260</y>
     <width>110</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Female</string>
   </property>
  </widget>
  <widget class="QLabel" name="pickupLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;</string>
   </property>
   <property name="text">
    <string>Pick Point :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PickupLineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>330</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="DestinationLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;</string>
   </property>
   <property name="text">
    <string>Destination :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="DestinationLineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>420</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="SelectCarLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>500</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;</string>
   </property>
   <property name="text">
    <string>Select Car :</string>
   </property>
  </widget>
  <widget class="QListWidget" name="SelectCarTypeWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>500</y>
     <width>361</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
